serde-redis-stream-derive/tests/it_compiles.rs

```diff
@@ -1,5 +1,5 @@
-use redis::streams::StreamKey;
-use serde_redis_stream::RedisStreamSerialize;
+use serde_redis_stream_derive::RedisStreamSerialize;
+use serde_redis_stream_interface::RedisStreamSerializable;
 
 #[derive(RedisStreamSerialize)]
```

Comment:
It would be really nice if we had more compilation tests. We probably want to have a compilation test for each feature (options, field serialization attributes)